go-controller/.gitignore

```diff
@@ -1,2 +1,4 @@
 _output
 _artifacts
+*.coverprofile
```

Comment:
Are these files generated by any of the make targets? I don't recall seeing them when I run some of the targets.

Reply:
that actually sneaked in from another place (these are the files generates by coveralls, but I didn't really want to include it)
nice catch :)